### DISCUSSION TITLE
Help: Failed to Load - NoTag: reader does not contain an id3 tag

### DISCUSSION THREAD

**Discussion by Funtimedave** [Marekkon5/onetagger#370] [Q&A]
Amazing tool - thank you so much for the work. 

Occasionally I get the following error when using quick tag feature on a folder as perr the title.  I say error I appreciate the program is probably working as intended and it is something my end.  I tried searching the discussions and could not see this listed.
Any idea how I can fix.  I tried to edit some fields but not sure which one to edit

**Comment by Marekkon5** [comment]
Hello, QT requires the title & artist tags to work, if you get this error, that means some of your tracks don't have any tags. You can insert title & artist tracks using Tag Edit or Auto Tagger.
